InAppSettingsKit/Models/IASKSpecifier.h

```diff
@@ -29,6 +29,8 @@
 - (id)initWithSpecifier:(NSDictionary *)specifier
         radioGroupValue:(NSString *)radioGroupValue;
 
+- (void)sortIfNeeded;
```

Comment:
Design-wise, I think it makes sense the way you wrote it. I wouldn't move it to the initializer.